setup.py

```diff
@@ -0,0 +1,32 @@
+import os
```

Comment:
It seems we are not actually pulling the files for some reason. I get this:
```
$ pip show net-analysis
Name: net-analysis
Version: 0.0.1
Summary: Network censorship detection tools
Home-page: https://github.com/Jigsaw-Code/net-analysis
Author: Jigsaw Operations, LLC
Author-email: fortuna@google.com
License: UNKNOWN
Location: /usr/local/google/home/fortuna/.local/share/virtualenvs/piptest-i3ZKsPr0/lib/python3.7/site-packages
Requires: 
Required-by: 
(piptest) fortuna@vini:~/firehook/piptest
$ pip show net-analysis -f
Name: net-analysis
Version: 0.0.1
Summary: Network censorship detection tools
Home-page: https://github.com/Jigsaw-Code/net-analysis
Author: Jigsaw Operations, LLC
Author-email: fortuna@google.com
License: UNKNOWN
Location: /usr/local/google/home/fortuna/.local/share/virtualenvs/piptest-i3ZKsPr0/lib/python3.7/site-packages
Requires: 
Required-by: 
Files:
  net_analysis-0.0.1.dist-info/INSTALLER
  net_analysis-0.0.1.dist-info/LICENSE
  net_analysis-0.0.1.dist-info/METADATA
  net_analysis-0.0.1.dist-info/RECORD
  net_analysis-0.0.1.dist-info/WHEEL
  net_analysis-0.0.1.dist-info/top_level.txt
```
The files are not actually there, I have no idea why.

Reply:
I wonder if this is because you're pulling from git and I ignored the distribution files

I just pushed the binary distribution files to the remote branch, try again and see if that works?

Reply:
I still don't get it:

```
$ pip show jigsaw-net-analysis -f
Name: jigsaw-net-analysis
Version: 0.0.1
Summary: Network analysis tools
Home-page: https://github.com/Jigsaw-Code/net-analysis
Author: Jigsaw Operations, LLC
Author-email: fortuna@google.com
License: UNKNOWN
Location: /usr/local/google/home/fortuna/.local/share/virtualenvs/piptest-i3ZKsPr0/lib/python3.7/site-packages
Requires: net-analysis
Required-by: 
Files:
  jigsaw_net_analysis-0.0.1.dist-info/INSTALLER
  jigsaw_net_analysis-0.0.1.dist-info/LICENSE
  jigsaw_net_analysis-0.0.1.dist-info/METADATA
  jigsaw_net_analysis-0.0.1.dist-info/RECORD
  jigsaw_net_analysis-0.0.1.dist-info/WHEEL
  jigsaw_net_analysis-0.0.1.dist-info/top_level.txt
```